src/google-analytics/GoogleAnalyticsService.js

```diff
@@ -0,0 +1,53 @@
+/**
+ * @implements {GoogleAnalyticsService}
+ * @memberof module:GoogleAnalytics
+ */
+class GoogleAnalyticsService {
+  constructor({ config }) {
+    this.analyticsId = config.GOOGLE_ANALYTICS_4_ID;
```

Comment:
That's actually the situation we're in right now. GA universal analytics is the current version and we can't just overwrite it because both the javascript snippet and identifier change with the upgrade. Because a handful of installs can continue using universal analytics longer than most of the community due to being 360 subscribers we need to be able to support both for a period of time. Once GA UA 360 reaches end of life we can deprecate and remove the old version.